python-sdk/src/astro/settings.py

```diff
@@ -68,10 +68,22 @@
     section=SECTION_KEY, key="load_table_autodetect_rows_count", fallback=1000
 )
 
-
 #: Reduce responses sizes returned by aql.run_raw_sql to avoid trashing the Airflow DB if the BaseXCom is used.
 RAW_SQL_MAX_RESPONSE_SIZE = conf.getint(section=SECTION_KEY, key="run_raw_sql_response_size", fallback=-1)
 
 # Temp changes
 # Should Astro SDK automatically add inlets/outlets to take advantage of Airflow 2.4 Data-aware scheduling
 AUTO_ADD_INLETS_OUTLETS = conf.getboolean(SECTION_KEY, "auto_add_inlets_outlets", fallback=True)
+
+LOAD_TABLE_SCHEMA_EXISTS = False
+
+
+def reload():
```

Comment:
No, they are not..!

This is not an issue in running tasks in Airflow because users don't usually processes that started before an environment variable change to get updated on the fly.

To only use `importlib.reload` on all the import path modules (e.g. `astro`, `astro.sql`, etc) was not enough to reload the values. Perhaps this is why we didn't have tests for checking if these settings were being applied.